.github/workflows/run_tests.yml

```diff
@@ -99,6 +99,40 @@ jobs:
       - name: Run tests
         run: go test -v -race ./...
 
+  tests_win:
+    name: Windows, Go
+    runs-on: windows-2022
```

Comment:
2022 seems to be too aggressive to me, it's not stable yet.

Reply:
Use 2019.

Reply:
Turns out that official golang images are either for WSC-22 or WSC-16. Image for WSC-16 is depricated, so I still use 2022 image.

Another option is to use base image WSC-19 and install Go, git, mingw and other necessary tools manually, it shouldn't be hard (except mingw), but I'd prefer to keep go image with WSC22 base.